Client/Cliqz/Frontend/Browser/BrowserViewControllerExtension.swift

```diff
@@ -291,6 +291,9 @@ extension BrowserViewController: ControlCenterViewDelegate {
     }
 
 	func showHint(type: HintType) {
+        if type == HintType.Antitracking && self.urlBar.isAntiTrackingButtonHidden(){
```

Comment:
`showHint` should be called only if the count is > 0
I don't know how this happen on the fresh tab
@naira-cliqz